rollout/bluegreen.go

```diff
@@ -223,6 +227,13 @@ func (c *RolloutController) syncRolloutStatusBlueGreen(previewSvc *corev1.Servic
 	oldRSs := roCtx.OlderRSs()
 	allRSs := roCtx.AllRSs()
 	newStatus := c.calculateBaseStatus(roCtx)
+
+	if replicasetutil.CheckPodSpecChange(r, newRS) {
+		roCtx.PauseContext().ClearPauseConditions()
+		roCtx.PauseContext().RemoveAbort()
+		newStatus = c.calculateRolloutConditions(roCtx, newStatus)
```

Comment:
We also can remove the calculateRolloutConditions method call here since we call it at https://github.com/argoproj/argo-rollouts/pull/314/files#diff-cfad87918ee0d3175ebd8e47f5efd159R270 before the persist command.

Reply:
After this change, we are good to merge!

Reply:
fixed in https://github.com/argoproj/argo-rollouts/pull/314/commits/4a076172096e501afa982e683ee83bcae892294d !